test/performance/io/format_fastq_benchmark.cpp

```diff
@@ -0,0 +1,176 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+// written for the purpose of the Bch Softwarepraktikum, early 2019 FU Berlin
+// @author: Clemens Cords <clemenscords@fu-berlin.de>
+
+#include <benchmark/benchmark.h>
+#include <cstring>
+#include <cmath>
+#include <chrono>
+#include <fstream>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
```

Comment:
new line between stl includes and seqan3 inlcudes. (Sorry for being so picky :D)